packages/ethereum-contracts/contracts/superfluid/Superfluid.sol

```diff
@@ -861,6 +877,18 @@ contract Superfluid is
                     valueForwarded ? 0 : msg.value,
```

Comment:
the boolean checking may not be necessary, simply forward all msg.value that's left is a safe and correct behaviour too.